### ISSUE TITLE
GUI: Filter bar clear filter button

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#983]
## As Is
The filter button used on all the pages in `Server Health` uses the letter `x` to visually indicate the purpose of the button. This is not screen reader friendly and not accessible.

## To Be
The filter button uses an icon that is `aria-hidden` and uses an `aria-label` attribute with the value set to `clear filter`

**Comment by dixsie** [comment]
Resolved by: https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-webui/+/25041
